### PR TITLE
Add initial models

### PR DESCRIPTION
**Pull request by adamskt**
Groups -> Postsm->Comments

### CODE REVIEW

File: SyncUp.Data/AppDbContext.cs [hunk 1]
```diff
@@ -64,6 +64,10 @@ public AppDbContext() { }
 
     public AppDbContext(DbContextOptions options) : base(options) { }
 
+    public DbSet<Group> Groups => Set<Group>();
+    public DbSet<Post> Posts => Set<Post>();
+    public DbSet<Comment> Comments => Set<Comment>();
```
Comment (MikaellaCroskrey-IntelliTect) [inline SyncUp.Data/AppDbContext.cs:69]: Should Events be here as well?

**Comment by adamskt** [comment]
Fises #2

**Review by MikaellaCroskrey-IntelliTect — APPROVED** [review]

File: SyncUp.Data/AppDbContext.cs (hunk 1 above)
Comment (adamskt) [reply SyncUp.Data/AppDbContext.cs:69]: SoonCatGIF

File: SyncUp.Data/Models/Group.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.IdentityModel.Tokens;
+
+namespace IntelliTect.SyncUp.Data.Models;
+
+public class Group : TenantedBase
+{
+    public long GroupId { get; init; }
+
+    [MaxLength(500)]
+    public required string Name { get; set; }
```
Comment (meghanmae) [inline SyncUp.Data/Models/Group.cs:10]: I have a preference to also add [Required] on top of these, but let's not block the PR for it.

**Pull request merged**